client/src/main/java/org/apache/celeborn/client/write/DataPusher.java

```diff
@@ -137,6 +138,7 @@ public void run() {
           }
         };
     pushThread.setDaemon(true);
+    pushThread.setUncaughtExceptionHandler(new ThreadExceptionHandler("DataPusher-" + taskId));
```

Comment:
nit:
Considering that the DataPusher thread is active on the client side, it may be appropriate to rename the thread with a prefix `celeborn`

Reply:
Done

Reply:
@cfmcgrady, IMO, the thread name does not need to give a prefix `celeborn`, otherwise other thread name should follow this pattern.

Reply:
> otherwise other thread name should follow this pattern.

Yes, I think all threads that are active on the client side should follow this pattern.

Reply:
@AngersZhuuuu, could you unify the thread names in another pull request?

Reply:
> @AngersZhuuuu, could you unify the thread names in another pull request?

Sure